### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (.btn-transparent)

### ISSUE THREAD

**Issue by brianteeman** [joomla-projects/j4adminui#63]
#### Issue
![image](https://user-images.githubusercontent.com/1296369/69328456-2cad7d00-0c47-11ea-8f83-908dc56a10e3.png)

`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.3/color-contrast?application=msftAI))
#### Target application
[Templates: Styles (Site) - kh - Administration](http://localhost/kh/administrator/index.php?option=com_templates&view=styles)
#### Element path
.btn-transparent
#### Snippet
`<strong class="text-success btn btn-transparent">
														<span class="icon-check-circle icon-md mr-2" area-hidden="true"></span> This is default template													</strong>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.83 (foreground color: #5fac23, background color: #ffffff, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 78.0.3904.97
